appservice/src/startStreamingLogs.ts

```diff
@@ -61,7 +58,17 @@ export async function startStreamingLogs(client: ISimplifiedSiteClient, verifyLo
 
                 await new Promise((onLogStreamEnded: () => void, reject: (err: Error) => void): void => {
                     let newLogStream: ILogStream;
-                    const logsRequest: request.Request = requestApi(`${client.kuduUrl}/api/logstream/${logsPath}`);
+
+                    const logsRequest: request.Request = request(`${client.kuduUrl}/api/logstream/${logsPath}`, {
```

Comment:
This kind of seems more error prone than using the kuduClient/request API to sign and format the request.  Any reason for the change?

Reply:
`signRequest` doesn't work like it did in the old sdk. Basically, if you want to use `signRequest`, you _have_ to send the request with a ServiceClient from "ms-rest-js". It's not compatible with the "request" module. That's a big part of why I added [createGenericClient](https://github.com/microsoft/vscode-azuretools/blob/master/ui/src/createAzureClient.ts#L32) in the UI package. I tried using that GenericClient here, but this log stream stuff is pretty unique, mostly because we keep the request open while the user is streaming logs. I plan to investigate this more in the future, because this it's pretty much the only code still using the "request" module and I'd love to remove that dependency (since it's deprecated)